lib/_emerge/post_emerge.py

```diff
@@ -88,6 +89,16 @@ def post_emerge(myaction, myopts, myfiles, target_root, trees, mtimedb, retval):
     settings = vardbapi.settings
     info_mtimes = mtimedb["info"]
 
+    # Do not update the mtimedb if we updated it recently.
+    if time.time() - mtimedb.lastUpdated() > 60 * 5:
```

Comment:
Does it affect updates of `resume` / `resume_backup` lists in mtimedb (which are quite important)?